Snakefile

```diff
@@ -46,6 +46,7 @@ ALL_FILES = [
     ('safetyLiabilities.json.gz', GS.remote(f"{config['TargetSafety']['outputBucket']}/safetyLiabilities-{timeStamp}.json.gz")),
     ('chemicalProbes.json.gz', GS.remote(f"{config['ChemicalProbes']['outputBucket']}/chemicalProbes-{timeStamp}.json.gz")),
     ('crispr_screens.json.gz', GS.remote(f"{config['CrisprScreens']['outputBucket']}/crispr_screens-{timeStamp}.json.gz")),
+    ('pharmacogenetics.json.gz', GS.remote(f"{config['Pharmacogenetics']['outputBucket']}/cttv012-{timeStamp}_pgkb.json.gz")),
```

Comment:
I agree. This was a shortcut I did because renaming would require changes in PIS.
I'll rename it properly in a different bucket and tell BE to change PIS

Reply:
solved in. https://github.com/opentargets/evidence_datasource_parsers/pull/185/commits/6f3d531ef14fb67fa632852a2e8f17af1b1fb05a